### PR TITLE
Dev/alha1.2.1

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -577,9 +577,7 @@
     <codeStyleSettings language="protobuf">
       <option name="RIGHT_MARGIN" value="80" />
       <indentOptions>
-        <option name="INDENT_SIZE" value="2" />
         <option name="CONTINUATION_INDENT_SIZE" value="2" />
-        <option name="TAB_SIZE" value="2" />
       </indentOptions>
```
Comment (TomKimsey) [inline .idea/codeStyles/Project.xml:581]: given the commit message im not sure why these are changed

File: src/main/java/com/hms_networks/americas/sc/extensions/system/http/SCHttpUtility.java [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  private static final String TEMP_RESPONSE_FILE_FOLDER = "/usr/http/";
+
+  /** Prefix appended to temporary files used to store HTTP responses. */
+  private static final String TEMP_RESPONSE_FILE_NAME_PREFIX = "sc_http_response_";
+
+  /** Counter used to uniquely identify temporary files used to store HTTP responses. */
+  private static int tempResponseFileNameCounter = 0;
+
+  /**
+   * Sets the http timeouts. Note: This changes the eWON's global HTTP timeouts and stores these
```
Comment (TomKimsey) [inline src/main/java/com/hms_networks/americas/sc/extensions/system/http/SCHttpUtility.java:46]: eWON should be Ewon or Ewon Flexy.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 # Ewon Flexy Extensions Library Changelog
 
+## Version 1.2.1
```
Comment (TomKimsey) [inline CHANGELOG.md:3]: Since this is new functionality shouldn't it be 1.3.0?

**Review by TomKimsey — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (alexjhawk) [reply CHANGELOG.md]: Corrected

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (alexjhawk) [reply .idea/codeStyles/Project.xml:581]: Added more commit description, although I am not 100% certain on the origin of these changes (the IDE did the changes)

**Review by TomKimsey — APPROVED** [review]

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (TomKimsey) [reply .idea/codeStyles/Project.xml:581]: That seems strange, ideally those would be a seperate commit.

Comment (alexjhawk) [reply .idea/codeStyles/Project.xml:581]: Updated

**Review by oliver-walker-hms — APPROVED** [review]

**Pull request merged**